String/test/ZFunction.test.js

```diff
@@ -0,0 +1,15 @@
+import ZFunction from '../ZFunction'
+
+test('Test Case 1', () => {
```

Comment:
Useless test case naming and useless `text` & `ZArray` variables.

Reply:
Improved naming and removed useless variables